ui/src/providers/i18n/locales/ja.json

```diff
@@ -46,6 +46,15 @@
   "Settings section 1": "",
   "Workspaces settings": "",
   "Workflow settings": "",
+  "Home": "",
```

Comment:
Any idea why ja.json is the only locales file accompanying en.json? 
We have french, spanish and chinese files as well. With `yarn i18n` it should update all of them 🤔

Reply:
I didn't notice. I have updated the code.